ui/profiles/sheets/ReferencersSheet.tsx

```diff
@@ -40,6 +40,7 @@ export default function Referencers({
 
       setUsers(users)
       setRefData(items[0].expand?.ref)
+      console.log('ReferencersSheet refData:', items[0].expand?.ref)
```

Comment:
This should be removed - we shouldn't check in any code that has `console.log` statements